public/app/features/dashboard/api/v1.ts

```diff
@@ -62,6 +63,10 @@ export class K8sDashboardAPI implements DashboardAPI<DashboardDTO, Dashboard> {
       obj.metadata.name = dashboard.uid;
       return this.client.update(obj).then((v) => this.asSaveDashboardResponseDTO(v));
     }
+    obj.metadata.annotations = {
+      ...obj.metadata.annotations,
```

Comment:
Do we also need to equip v0 with this? Couldn't see it anywhere (but I'm on my phone, so could've missed it)

Reply:
as discussed offline